content/influxdb/v2.0/write-data/best-practices/schema-design.md

```diff
@@ -0,0 +1,229 @@
+---
+title: InfluxDB schema design
+description: >
+  Improve InfluxDB schema design and data layout. Store unique values in fields and other tips to reduce high cardinality in InfluxDB and make your data more performant.
+menu:
+  influxdb_2_0:
+    name: Schema design
+    weight: 201
+    parent: write-best-practices
+---
+
+Each InfluxDB use case is unique and your [schema](/influxdb/v2.0/reference/glossary/#schema) design reflects that uniqueness. Discover a few design guidelines that we recommend for most use cases:
```

Comment:
...the uniqueness. We recommend the following design guidelines for most use cases: